src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -299,6 +305,59 @@ public CommandBase driveToNearest() {
 
     }
 
+    public CommandBase driveUntilTipped() {
+        return cmd().onExecute(() -> {
+            move(0.0, 0.25, 0.0);
+
+        }).runsUntil(() -> Math.abs(pigeonGyro.getPitch()) > 9);
+    }
+
+    public CommandBase balance() {
+
+        PersistenceCheck balancedCheck = new PersistenceCheck(15, () -> Math.abs(pigeonGyro.getPitch()) < 7);
```

Comment:
We should tweak this value, 7 is probably too high. And it should be using the combined pitch/roll value based on the robot's yaw.